sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -815,6 +815,24 @@ class Dataset[T] private[sql](
     println(showString(numRows, truncate, vertical))
   // scalastyle:on println
 
+  /**
+   * Returns the default number of rows to show when the show function is called without
+   * a user specified max number of rows.
+   * @since 2.3.0
```

Comment:
not needed as this is private, moreover, it'd be since 2.4.0

Reply:
I'll update this.